.github/workflows/license-checker.yml

```diff
@@ -3,10 +3,10 @@ name: License checker
 on:
   push:
     branches:
-      - master
+      - planner_refactory
   pull_request:
     branches:
-      - master
+      - planner_refactory
```

Comment:
modify for license checker.
I'll revert to `master` at the last pr of branch `planner refactory`